### ISSUE TITLE
Make app compatible with Voice Over

### ISSUE THREAD

**Issue by lemoness** [HabitRPG/habitica-android#180]

**Comment by AlgoRyan** [comment]
This is something that I have some experience with. Are there any particular requirements? How many users want this?

**Comment by lemoness** [comment]
This has been requested on several occasions. As for the requirements, perhaps @viirus has some thoughts.

**Comment by anitawoodruff** [comment]
I am happy to take a look at this. Stuff I plan to check:
- content descriptions - sensible labels are read out for UI elements when users explore by touch with Talkback. Purely decorative items are not read out at all.
- content grouping - adjacent content is read out together where it makes sense, but not when it doesn't.
- focus order - the right elements are focusable (purely decorative elements do not need to be focusable), so they are read out in a sensible order when the user swipes through them with Talkback enabled.
- live regions - changes to view triggered by user actions on other elements are announced, politely

Let me know if there's anything I'm missing!

**Comment by phillipthelen** [comment]
@anitawoodruff That sounds good to me!
